### DISCUSSION TITLE
Unable to install theia dependencies

### DISCUSSION THREAD

**Discussion by DayronDLR** [eclipse-theia/theia#12177] [General]
I'm trying to install dependencies for the Theia editor but I'm not able do.

**Steps to Reproduce:**

1. git clone https://github.com/TypeFox/theia-xtext-sprotty-example.git
2. cd language-server  && ./gradlew build -> this is ok.
3. cd ../theia && yarn -> this failed compile.

**Error:**

![image](https://user-images.githubusercontent.com/37316384/218355195-9c4445cc-eed9-4fc2-95f9-6a9d930b96c2.png)

**However the prerequisites said this:**

![image](https://user-images.githubusercontent.com/37316384/218355264-c063f838-a170-4207-acc7-454ff2382820.png)

**Systems**

MacOs and Linux, none of them work. The result is the same.

**Comment by msujew** [comment]
Hey @DayronDLR,

the prerequites of Theia are updated for the current version of Theia (1.34.0). The version used in the example repo is 3 years old by now and heavily outdated. So outdated in fact, that the API Theia is using to speak directly with the language server doesn't exist anymore.

Instead of connecting Theia straight to the language server, we now recommend developers to build vscode extensions for their languages and install those into Theia. For a comparable feature set, take a look at [`sprotty-vscode`](https://github.com/eclipse-sprotty/sprotty-vscode) and their examples (an xtext example is included).
